addons/website/static/src/scss/website.edit_mode.scss

```diff
@@ -81,14 +81,34 @@ $-editor-messages-margin-x: 2%;
     }
 
     &.oe_structure.oe_empty, &[data-oe-type=html], .oe_structure.oe_empty {
-        &#wrap:empty, &#wrap > .oe_drop_zone.oe_insert:not(.oe_vertical):only-child {
+        &#wrap:empty, &#wrap > .oe_drop_zone.oe_insert:not(.oe_vertical) {
             @extend %o-editor-messages;
-            padding: 112px 0px;
+
+            &:only-child {
+                padding: 112px 0px;
+            }
+
+            &:not(:only-child) {
+                padding: 0px;
+                &::before {
+                    font-size: 18px;
+                    line-height: 25px!important;
+                }
+            }
         }
-        > .oe_drop_zone.oe_insert:not(.oe_vertical):only-child {
+        > .oe_drop_zone.oe_insert:not(.oe_vertical) {
```

Comment:
It does not appear code-wise but a side effect of this PR:

Before:
![image](https://user-images.githubusercontent.com/10338094/223513845-7c573623-9b55-4ac2-b34c-5c8526b1a028.png)

After:
![image](https://user-images.githubusercontent.com/10338094/223513739-5ec3d4bb-5721-4ac6-a5b6-8df9cd6cae11.png)

=> Spacing around the dropzone. I visually like it (kinda the same to me) but functionally I find this very bad: the page "jumps" when toggling the dropzones because they now take spacing in the page (in the example above, the first snippet is now lower than before toggling the dropzones). I'd really prefer to keep the fact that toggling the dropzones just make them appear as an overlay of the page, without making snippets move around... and once we do that, I am not sure that displaying the text will look nice :grimacing: (to check)

cc @pierre-pde

Reply:
@qsm-odoo  & @rdeodoo  IMO, the "jump" is not so problematic because, if you decide to drop the block there, there will be a jump.
But I understand that it's not ideal, let see how it looks like without the dropzone taking space.

Reply:
IMO, there is no relation at all between the "jump" post-drop and the "jump" pre-drop :grimacing: Here the problem is that you scroll somewhere to add a block, start dragging and the whole screen changes because blocks are being pushed towards the bottom by `~100px * number of snippets above them`.

IMO it's not "not ideal", it's a no-go. Here an example with some snippets (I used multiples number snippets) + a media list + a numbers + a banner:
![Screenshot from 2023-03-08 14-36-13](https://user-images.githubusercontent.com/10338094/223727456-47c08cea-0db3-449b-80e0-705ac68c3cce.png)

=> I want to add something between the number and the banner (location which is visible on the screen), I start dragging, I jump to this:
![Screenshot from 2023-03-08 14-36-26](https://user-images.githubusercontent.com/10338094/223727783-26185711-6832-47ca-92d1-d4c44e62a8be.png)

=> My location is gone I have to scroll (up? down?) to find it again.

Also small side-effect: the auto-scroll after drop is actually slightly flickering because of this too: the dropzones taking space are being removed during the smooth scroll making the animation not as smooth in some cases.

Reply:
Ok, of course the effect that user is being pushed towards the bottom `by ~100px * number of snippets above them` is not acceptable, I agree

Reply:
Note that it's not `by ~100px * number of snippets above them` but `by ~100px * number of visible at screen snippets above them`.
But indeed good catch it's really terrible in those proportions

Reply:
> Note that it's not by ~100px * number of snippets above them but by ~100px * number of visible at screen snippets above them.

Definitely not, cfr my screenshots above explaining the issue: there is jump of more than one screen height, while only 2-3 snippets are visible on the screen :wink:

Reply:
Would you both mind having a look at the runbot and tell me what you think?
- Don't pay attention to the code
- Don't pay attention to the dropzone text style (color/bg color), it's a wip, I still have to find something which works on every color. It basically was an issue we already have as reported by the india testing: go to /forum and drag & drop snippet in the Welcome message, same issue.
- So just functionally about the 5 use cases:
  - Empty website.page in edit mode (no drag & drop)
  - Empty website.page with drag & drop
  - Non empty website.page with drag & drop
  - Empty product description with drag & drop
  - Non empty product description with drag & drop

If that sounds reasonable, I'll fine tune my ~crap~ wip

Reply:
@rdeodoo , I had a look, the UI looks great to me.
I don't like the fact the placeholder has a big font-weight, but I know we're not supposed to judge style.

Reply:
@rdeodoo I closed my eyes and tested and it looks great :smile: ... joke aside I am honestly confused on what you want me to check: it's like before but with the text (which is a nice idea) but I can't judge the text style... so in the end... I can't say much right? :smile: 

Maybe just a suggestion: maybe there is no need to display the text for the standard use case? When the text is "Drag and drop a building block here" ? That way that would lighten the UI by a lot while still providing what you are trying to fix: being able to see which dropzones are common to some pages in some cases? And that would reduce the need to make this text perfectly visible in all cases?

Reply:
I am not sure how we could differentiate a base case from another case tho?
We can of course differentiate the website.page case from the product case but even in the product case there is a kind of "base case" which is the product description field and a more complexe case with the "Share it here etc..".

Reply:
The default message is automatically added in JS as the `data-editor-message` attribute (something like that). That JS could also put something like `data-is-default-editor-message="true"` so you can differentiate in CSS.